doc: samples: dect: add note about RF use (#15928)
nrfconnect/sdk-nrf: samples/dect/dect_phy/hello_dect/README.rst

   DECT NR+ operates on free but regulated radio channels.
   The regulations and availability of the channels vary by country and region.
   It is your responsibility to operate the devices according to local regulations, both at the development site and in the device operating regions.
   If you are in the EU or US and are permitted to operate on the DECT band and can ensure that access rules are met, you can use the ``overlay-eu.conf`` and ``overlay-us.conf`` Kconfig overlays, respectively.
   If other configuration is required, set the carrier using the :ref:`CONFIG_CARRIER <CONFIG_CARRIER>` Kconfig option, and the transmission power using the :ref:`CONFIG_TX_POWER <CONFIG_TX_POWER>` Kconfig option.

>    If you as a user are't permitted to operate on the DECT band or can't ensure that access rules are met, you are responsible to ensure there are no radio emissions from the devices running the sample.

Reviewer: ```suggestion
   If you, as a user, are not permitted to operate on the DECT band or cannot ensure that access rules are met, you must ensure there are no radio emissions from the devices running the sample.
```

>    This can for instance be done with the use of an RF chamber.

Reviewer: ```suggestion
   This can be done with the use of an RF chamber.
```
